pkg/ecctl/init.go

```diff
@@ -67,6 +73,13 @@ const (
 )
 
 var (
+	hostChoiceMsg = `
```

Comment:
I'm sure you already thought about it but do the `govcloud` and `essp` env require a different approach during our init flow?

Reply:
I had thought to change this afterwards but might as well do it now :)